community/kernel/src/test/java/org/neo4j/kernel/impl/util/ConvertersTest.java

```diff
@@ -125,6 +128,53 @@ public void emptyOptionalWhenOnlyPort()
         assertFalse( hostname.isPresent() );
     }
 
+    @Test
+    public void ipv6Works()
+    {
+        // with
+        String full = "1234:5678:9abc:def0:1234:5678:9abc:def0";
+        List<Pair<String,OptionalHostnamePort>> cases = Arrays.asList(
+                Pair.of( "::/0", new OptionalHostnamePort( "::/0", null, null ) ),
```

Comment:
Why are we testing address block notation? Where does the need for this come from?